docs/api/field_type_reference.md

```diff
@@ -1501,6 +1501,157 @@ new FieldType\Value([
 ]);
 ```
 
+### GraphQL Field Type operations
+
+To get a Matrix Field Type, you will need to specify a Content ID, a Content Type, and a Field Type. 
+The types used for input and returned are named after the Type and the Field: 
+
+- `{TypeIdentifier}{FieldIdentifier}RowInput`
+- `{TypeIdentifier}{FieldIdentifier}Row`
+
+The example below shows GraphQL query for a List Content Type (Content Type with a Matrix Field Type added), that has two fields:
+
+- `name`: `ezstring`
+- `ingredientList`: `ezmatrix` with two columns: `ingredient` and `quantity`
+
+```
+{
+  content {
+    #provide content ID, location ID or remote ID
+    list(id: 123) {
+      name
+      ingredientList {
+        ingredient
+        quantity
+      }
+    }
+  }
+}
+```
+
+The Type returned for the Matrix Field exposes columns defined in the Field definition:
+
+```
+{
+  "data": {
+    "content": {
+      "list": {
+        "name": "Cake ingredients",
+        "ingredientList": [
+          {
+            "ingredient": "Butter",
+            "quantity": "200 grams"
+          },
+          {
+            "ingredient": "Sugar",
+            "quantity": "100 grams"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+**Query for the Field Type and Field definition's details**
+
+```
+{
+  content {
+    _types {
+      list {
+        ingredientList {
+          settings {
+            minimumRows
+            columns {
+              name
+              identifier
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Response should list the exposed Field Type settings:
+
+- minimumRows
+- columns
+    - name
+    - identifier
+
+Example response:
+
+```
+{
+  "data": {
+    "content": {
+      "_types": {
+        "list": {
+          "ingredientList": {
+            "settings": {
+              "minimumRows": 1,
+              "columns": [
+                {
+                  "name": "ingredient",
+                  "identifier": "ingredient"
+                },
+                {
+                  "name": "quantity",
+                  "identifier": "quantity"
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+**Mutation**
+
+To create a Matrix Field Type you need to define Field Type and Field definition identifiers.
+
+Below example shows you how to create a List Content Type (Content Type with a Matrix Field Type added) that has two fields:
```

Comment:
I wouldn't call it "create a List Content Type". First, we should really rename the type used as an example to something meaningful, not a set of technical terms. "Recipe" with an "ingredients" field would really be a better choice. "nutritionFacts" is also a good one (calories, etc). Then the sentence would be to "how to create a recipe content item".

Note: you can NOT create and edit content types with GraphQL. Only content items.